concepticondata/conceptlists/Monnier-2014-1031.tsv

```diff
@@ -11,17 +11,17 @@ Monnier-2014-1031-9	9	agréable	pleasurable			ADJ			8.28571428571429	5.038961038
 Monnier-2014-1031-10	10	aide	help	HELP (ASSISTANCE)	3467	NOUN			7.08108108108108	4.81081081081081	7.17647058823529	5	7	4.65	4	2	1	171.41	52.3	2.95
 Monnier-2014-1031-11	11	aigle	eagle	EAGLE	1905	NOUN	AF	82	6.52631578947368	5.02631578947368	6.58823529411765	4.76470588235294	6.47619047619048	5.23809523809524	5	3	1	5.5	7.91	6.77
 Monnier-2014-1031-12	12	aiguille	needle	NEEDLE (FOR SEWING)	1382	NOUN	AF	154	3.9	4.34	4.27777777777778	4.77777777777778	3.6875	4.09375	8	5	2	10.4	18.38	6.53
-Monnier-2014-1031-13	13	ail 	garlic	GARLIC	2368	NOUN	B	4	4.21621621621622	3.67567567567568	4.88235294117647	3.47058823529412	3.65	3.85	3	2	1	9.14	7.97	6.34
-Monnier-2014-1031-14	14	aile 	wing	WING	1257	NOUN	B	5	6.05405405405405	4.40540540540541	5.82352941176471	3.58823529411765	6.25	5.1	4	2	1	15	20.47	6.32
+Monnier-2014-1031-13	13	"ail "	garlic	GARLIC	2368	NOUN	B	4	4.21621621621622	3.67567567567568	4.88235294117647	3.47058823529412	3.65	3.85	3	2	1	9.14	7.97	6.34
```

Comment:
Again cases of `"` here. All should be deleted, spaces before and after tabstop as well.